### DISCUSSION TITLE
OpenTelemetry .NET Logs Serilog integration

### DISCUSSION THREAD

**Discussion by andrei-micuda** [open-telemetry/opentelemetry-dotnet#3905] [Q&A]
Hello!

I have been following the (getting started guide)[https://github.com/open-telemetry/opentelemetry-dotnet/tree/main/docs/logs/getting-started] for OpenTelemetry Logs in .NET that is using `Microsoft.Extensions.Logging`.

My question is, can I integrate the same behaviour with `Serilog`? I found a [Serilog extention proposal](https://github.com/open-telemetry/opentelemetry-dotnet/tree/main-logs/src/OpenTelemetry.Extensions.Serilog) that seems to achieve what I am looking for, but from my understanding that is far from being released. Do I have any other options as of today?

Thank you!

**Comment by Depechie** [comment]
for ref: https://github.com/open-telemetry/opentelemetry-dotnet/discussions/3857
